pkg/dynamic/helpers.go

```diff
@@ -0,0 +1,263 @@
+/*
+Copyright 2021 The Tekton Authors
+
```

Comment:
v minor: I'd rename the file and call this reconcile.go or something. helpers.go is ambiguous.

Reply:
renamed to `custom_reconcile.go` as the changes added in that functions are related to `customResource`